src/main/java/edu/uci/ics/crawler4j/crawler/Page.java

```diff
@@ -122,7 +122,7 @@ public void load(HttpEntity entity) throws Exception {
       contentCharset = charset.displayName();
     }
 
-    contentData = EntityUtils.toByteArray(entity);
+    contentData = EntityUtils.toString(entity, "utf-8").getBytes();
```

Comment:
Maybe we can use https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html instead of the hard-coded `utf-8` string